src/components/Home.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './home.css';
+
+function Home() {
+  return (
+    <div>
```

Comment:
- [ ] As required please make sure that you follow the general layout should match the given wireframe.
- [ ] Kindly add the `Math Magicians` logo to the left of the header.
- [ ] Kindly make sure that your Navigation bar is aligned to the right as seen in the given wireframe.
- [ ] The text `Welcome To The Homepage` should be aligned to the left and not centered as seen in the given wireframe
- [ ] Your paragraph on the page should be aligned to the left as seen in the given wireframe.

|   |  |
| ------------- | ------------- |
| Your Design | <img width="917" alt="image" src="https://github.com/roodjinocherilus/magicmath/assets/52934128/88ffcc2d-740e-45fc-ba79-9139b1f9e491">  |
| Given Wireframe  | ![image](https://github.com/roodjinocherilus/magicmath/assets/52934128/f9125fa6-5a72-47e4-9026-f4ca7100fea9)  |